srcpkgs/ghc/template

```diff
@@ -39,6 +39,19 @@ desc_option_bindist="Create a binary distribution"
 # GHC's bundled tarball is from 2017, buggy on some archs
 _ffi_rev=4d6d2866ae43e55325e8ee96561221804602cd7a
 
+case "$XBPS_TARGET_MACHINE" in
+aarch64)
+        hostmakedepends+=" llvm"
+        depends+=" llvm"
+        ;;
+aarch64-musl)
+        # does not work with gold
+        configure_args+=" --disable-ld-override"
+        hostmakedepends+=" llvm"
+        depends+=" llvm"
+        ;;
+esac
```

Comment:
```suggestion
configure_args+=" --disable-ld-override"
case "$XBPS_TARGET_MACHINE" in
aarch64*)
        hostmakedepends+=" llvm"
        depends+=" llvm"
        ;;
esac
```

Reply:
Why does only aarch64 require llvm? Could it be made to work with llvm for other targets?

Reply:
Other targets have a native backend, which is way faster than llvm. (Rebuilding ghc takes almost 24 hours on my pinebook.)